src/datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -343,7 +343,7 @@ def _convert_label_categories(self, obj):
             converted["label_groups"].append(
                 {
                     "name": cast(label_group.name, str),
-                    "group_type": cast(label_group.group_type, str),
+                    "group_type": cast(label_group.group_type, int),
```

Comment:
For the backward compatibility of Datumaro data format, it seems we serialize it with string as same as before. And, do we need to update the import part also?

Reply:
```python
class GroupType(IntEnum):
    EXCLUSIVE = 0
    INCLUSIVE = 1
    RESTRICTED = 2

    def to_str(self) -> str:
          ...

    @classmethod
    def from_str(cls, text: str) -> GroupType:
          ...
```

Reply:
Thanks. I added at fe740c48f62a2808dc5157c756e23ac29cc28984